<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartWindow</class>
 <widget class="QMainWindow" name="ChartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>598</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Chart - Spire</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources/Resources.qrc">
    <normaloff>:/icons/chart_line.png</normaloff>:/icons/chart_line.png</iconset>
  </property>
  <widget class="QWidget" name="m_centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="Spire::UI::ScalableScrollBar" name="m_verticalScrollBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Spire::ChartPlotView" name="m_chart" native="true">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="Spire::UI::ScalableScrollBar" name="m_horizontalScrollBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="m_toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_panAction"/>
   <addaction name="m_zoomAction"/>
   <addaction name="separator"/>
   <addaction name="m_autoScaleAction"/>
   <addaction name="m_lockGridAction"/>
   <addaction name="m_linkAction"/>
  </widget>
  <action name="m_panAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/icons/pan_view.png</normaloff>:/icons/pan_view.png</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="m_zoomAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/icons/magnifying_glass.png</normaloff>:/icons/magnifying_glass.png</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="toolTip">
    <string>Zoom In/Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="m_autoScaleAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/icons/fit_to_size.png</normaloff>:/icons/fit_to_size.png</iconset>
   </property>
   <property name="text">
    <string>Auto Scale</string>
   </property>
   <property name="toolTip">
    <string>Enable Auto-Scaling</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="m_lockGridAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/icons/lock_view.png</normaloff>:/icons/lock_view.png</iconset>
   </property>
   <property name="text">
    <string>Lock Grid</string>
   </property>
   <property name="toolTip">
    <string>Locks the grid.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="m_linkAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/icons/link.png</normaloff>:/icons/link.png</iconset>
   </property>
   <property name="text">
    <string>Link</string>
   </property>
   <property name="toolTip">
    <string>Link to another window.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Spire::ChartPlotView</class>
   <extends>QWidget</extends>
   <header>Spire/Charting/ChartPlotView.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Spire::UI::ScalableScrollBar</class>
   <extends>QWidget</extends>
   <header>Spire/UI/ScalableScrollBar.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
